### DISCUSSION TITLE
Have a table/form field depend on some addition to URL.hash

### DISCUSSION THREAD

**Discussion by zischwartz** [nocodb/nocodb#1371] [Features]
Looking for a way to send essentially the same form to different groups of people and have the correct group recorded in the table. A similar use-case as [airtable's prefilling a form](https://support.airtable.com/hc/en-us/articles/234982508-Prefilling-a-form) feature except ideally the field would not be visible to the user filling out the form. Adding to the url.hash seems to make sense, but if anyone has any direction or ideas, please let me know, I'm totally new to this codebase, so any guidance is appreciated.

Semi related to #1245 as that would be a worst case way to achieve a similar end result.

**Comment by dstala** [comment]
@zischwartz Right now, NocoDB does not support prefilling a form (neither for default values- nor via query parameters). Since we do not have UI-based mechanisms for duplicating tables, it is going to be a manual effort of creating tables as required.  

Multi-select option for the group could help provide the right options, but that's not what you are looking for I believe.

**Comment by bonacci-johannes** [comment]
this would be a great feature!
